tests/sources/test_jira.py

```diff
@@ -418,7 +474,7 @@ async def test_verify_projects():
 
 @pytest.mark.asyncio
 async def test_verify_projects_with_unavailable_project_keys():
-    """Test _verify_projects method with unavaiable project keys"""
+    """Test _verify_projects method with unavailable project keys"""
```

Comment:
```suggestion
```

nit: you can simply remove that docstring as the method name already contains all the information needed